spec/game_spec.rb

```diff
@@ -1,66 +1,12 @@
-require 'spec_helper'
+require_relative 'spec_helper'
```

Comment:
see why you did this. private and not supposed to be writing for user input right now.